### PR TITLE
Add loyalty points

### PR DESCRIPTION
**Pull request by mickstar**
@indyfromoz

### CODE REVIEW

**Review by indyfromoz — APPROVED** [review]
🚀

File: src/main/java/au/com/dmg/fusion/response/paymentresponse/LoyaltyAccountID.java [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+        private EntryMode entryMode;
+        private IdentificationType identificationType;
+        private IdentificationSupport identificationSupport;
+        private String loyaltyID;
+
+        public Builder() {
+        }
+
+        Builder(EntryMode entryMode, IdentificationType identificationType, IdentificationSupport identificationSupport, String loyaltyID) {
+            this.entryMode = entryMode;
```
Comment (mickstar) [inline src/main/java/au/com/dmg/fusion/response/paymentresponse/LoyaltyAccountID.java:79]: identification type not written

File: src/main/java/au/com/dmg/fusion/response/paymentresponse/LoyaltyAcquirerData.java [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    public static class Builder {
+        private String loyaltyAcquirerID;
+        private String approvalCode;
+        private SaleTransactionID loyaltyTransactionID;
+        private String hostReconciliationID;
+
+        public Builder() {
+        }
+
+        Builder(String loyaltyAcquirerID, String approvalCode, SaleTransactionID loyaltyTransactionID, String hostReconciliationID) {
```
Comment (mickstar) [inline src/main/java/au/com/dmg/fusion/response/paymentresponse/LoyaltyAcquirerData.java:69]: SaleTransactionID type is used for parameter "loyaltyTransactionID"

File: src/main/java/au/com/dmg/fusion/response/paymentresponse/LoyaltyAcquirerData.java [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+            this.loyaltyAcquirerID = loyaltyAcquirerID;
+            return Builder.this;
+        }
+        
+        public Builder approvalCode(String approvalCode){
+            this.approvalCode = approvalCode;
+            return Builder.this;
+        }
+
+        public Builder loyaltyTransactionID(SaleTransactionID loyaltyTransactionID){
```
Comment (mickstar) [inline src/main/java/au/com/dmg/fusion/response/paymentresponse/LoyaltyAcquirerData.java:86]: probably wrong type here

**Pull request merged**